src/cmake/axom-config.cmake.in

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+# other Axom Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+#------------------------------------------------------------------------------
+
+cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+
+@PACKAGE_INIT@
+
+if(NOT AXOM_FOUND)
+
+  #----------------------------------------------------------------------------
+  # Set version and paths
+  #----------------------------------------------------------------------------
+  
+  set(AXOM_VERSION       "@AXOM_VERSION_FULL@")
+  set(AXOM_VERSION_MAJOR "@AXOM_VERSION_MAJOR@")
+  set(AXOM_VERSION_MINOR "@AXOM_VERSION_MINOR@")
+  set(AXOM_VERSION_PATCH "@AXOM_VERSION_PATCH@")
+  
+  set(AXOM_INSTALL_PREFIX "@AXOM_INSTALL_PREFIX@")
+  set(AXOM_INCLUDE_DIRS "${AXOM_INSTALL_PREFIX}/include")
+
+  #----------------------------------------------------------------------------
+  # Set user configuration options and features
+  #----------------------------------------------------------------------------
+
+  # Language features
+  set(AXOM_ENABLE_FORTRAN     "@ENABLE_FORTRAN@")
+  set(AXOM_USE_CUDA           "@AXOM_USE_CUDA@")
```

Comment:
We should probably publish `AXOM_CUDA_ARCH` here.  That way, downstream build systems can access that information and add sanity checks etc. This would help with #236

Reply:
Let's handle that separately, once we settle on #236, and where this goes.

Reply:
I think irrespective of where the `CUDA_ARCH` is handled and where the value of `CUDA_ARCH` is coming from, we should have it in the `cmake-config`. At this point, we only have this information in `AXOM_CUDA_ARCH`. This can change in the future, but, I think it is useful to have this in the cmake-config, so that it can be made accessible in the build system of downstream libraries.

Please, create a Github issue regarding publishing `CUDA_ARCH` in our CMake config.

Reply:
See #242. Please, feel free to make additional comments / suggestions.